src/main/java/com/teragrep/pth_06/jooq/generated/journaldb/tables/Logfile.java

```diff
@@ -226,7 +226,7 @@ public Schema getSchema() {
 
     @Override
     public List<Index> getIndexes() {
-        return Arrays.<Index>asList(Indexes.LOGFILE_BUCKET_ID, Indexes.LOGFILE_CATEGORY_ID, Indexes.LOGFILE_CIX_LOGFILE_HOST_ID_LOGTAG_LOGDATE, Indexes.LOGFILE_CIX_LOGFILE_LOGDATE_HOST_ID_LOGTAG, Indexes.LOGFILE_HOST_ID, Indexes.LOGFILE_IX_LOGFILE_EXPIRATION, Indexes.LOGFILE_IX_LOGFILE__SOURCE_SYSTEM_ID, Indexes.LOGFILE_PRIMARY, Indexes.LOGFILE_UIX_LOGFILE_OBJECT_HASH);
```

Comment:
index missing in generation, why?

Reply:
`git log -S LOGFILE_CIX_LOGFILE_HOST_ID_LOGTAG_LOGDATE --source --all` told me it is internal commit `d465ecf71a17cac7d165c21a6570e51f03c19e28` (tag 2.2.0) that removed it